.github/CODEOWNERS

```diff
@@ -239,7 +239,7 @@
 # ServiceOwners:                                                   @olduroja
 
 # PRLabel: %Cognitive - Language
-/sdk/cognitivelanguage/                                            @quentinRobinson
+/sdk/cognitivelanguage/                                            @quentinRobinson @bidisha-c @mikaelsitruk @sarkar-rajarshi @ahmedaashour
```

Comment:
The following folks need to make their `Microsoft` organization membership public:

- mikaelsitruk
- ahmedaashour